sway-fmt-v2/src/utils/helpers.rs

```diff
@@ -0,0 +1,37 @@
+use crate::Formatter;
+use sway_parse::Ty;
+use sway_types::Spanned;
+
+/// Handling of a ty, currently this does not apply any formatting.
+/// It simply pushes the string version of the ty.
+pub fn handle_ty(ty: &Ty, push_to: &mut String) {
+    push_to.push_str(ty.span().as_str());
+}
+
+/// Handles bracket open scenerio. Checks the config for the placement of the bracket.
+/// Modifies the current shape of the formatter.
+pub fn handle_bracket_open(push_to: &mut String, formatter: &mut Formatter) {
```

Comment:
small nit but maybe make this `handle_open_bracket` & `handle_close_bracket` instead